packages/paste-website/src/components/site-wrapper/sidebar/Header.tsx

```diff
@@ -3,8 +3,8 @@ import {Text} from '@twilio-paste/text';
 import styled from '@emotion/styled';
```

Comment:
I moved a bunch of files related to `site-wrapper` into that folder to make it easier to work on this stuff.